spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -475,7 +484,7 @@ def contextMenuEvent(self, event):
         """Reimplement Qt method"""
         self.menu.popup(event.globalPos())
         event.accept()
-        
+

```

Comment:
@juanis2112 please revert this and the other style changes you made after this line (white/blank spaces), thanks :)